sequoia/methods/aux_tasks/ewc.py

```diff
@@ -64,10 +64,10 @@ class Options(AuxiliaryTask.Options):
         # that hyper-parameter here.
         coefficient: float = uniform(0.0, 100.0, default=1.0)
         # Batchsize to be used when computing FIM (unused atm)
-        batch_size_fim: int = 64
+        batch_size_fim: int = 32
         # Number of observations to use for FIM calculation
-        sample_size_fim: int = 400
-        # Fisher information representation type  (diagonal or block diagobnal).
+        sample_size_fim: int = categorical(2, 4, 8, 16, 32, 64, 128, 256, 512, default=8)
```

Comment:
This `sample_size_fim` is the total number of samples to use to calculate the FIM used in EWC.. As such I think these options are too low!

Reply:
I think that EWC scales really badly w.r.t. this. At least that what I understood from @oleksost

Reply:
Edit: This is the number of batches, not the number of samples. Therefore this might make sense, altough the range is still pretty huge. Marking this as resolved for now.